lib/index.js

```diff
@@ -186,6 +192,7 @@ export default class PhoneInput extends Component {
             editable={!disabled}
             autoCorrect={false}
             style={[styles.text, this.props.textStyle]}
+            onSubmitEditing={() => this.onSubmitEditing()}
```

Comment:
This will eat the `onSubmitEditing` event, which could be useful to some users. Please make sure that the event object is properly passed to the provided `onSubmitEditing` handler.